### ISSUE TITLE
FluffyApp Compatibility

### ISSUE THREAD

**Issue by philip-peterson** [posativ/regenwolken#41]

**Comment by posativ** [comment]
I've opened an issue in their forum a year ago, but I got no reply. Somehow FluffyApp does not completely rely on the REST API but rather uses some (not required) session ids, that I do not provide and will not provide unless I exactly know what FluffyApp requires.

**Comment by philip-peterson** [comment]
I recently checked up on FluffyApp and found that it's somehow been [made an official component](https://www.getcloudapp.com/blog/CloudApp-comes-to-windows) of the CloudApp system. Has the compatibility situation changed at all, by chance?

**Comment by posativ** [comment]
Yes, the situation changed to a new API and enforced SSL which requires a custom root CA to create a certificate for their new SSL-enabled API endpoint. I did not upgrade regenwolken to the new API and do not have plans to implement it in the future.

**Comment by philip-peterson** [comment]
Oh.. so regenwolken won't even work with the newer Mac versions of CloudApp, then?

**Comment by posativ** [comment]
Unfortunately, not. You can still use the old CloudApp (~1.5).
